pkg/github/client.go

```diff
@@ -139,5 +140,10 @@ func (gh *Github) ListPullRequests() ([]*gogithub.PullRequest, error) {
 
 var (
 	printTerminal = terminal.Output
-	readTerminal  = terminal.Read
+	readTerminal  = func() (string, error) {
+		var input string
+		_, err := fmt.Scanln(&input)
+
+		return input, err
+	}
```

Comment:
why do you remove the usage of the terminal package?

Reply:
`scanLn` can't scan type `interface{}` which broke the whole flow of accepting input